pages/formester-vs-typeform.vue

```diff
@@ -450,3 +519,13 @@ thead th {
   }
 }
 </style>
+<style>
+.present{
+  color: #6BC8AD;
+  font-weight: 600
+}
+.absent{
+  color: #F35E54; 
+  font-weight: 600
+}
```

Comment:
same here, just check for the places where they are present and keep one at the best place 🥇